tests/ondemand/ondemand_document_stream_tests.cpp

```diff
@@ -0,0 +1,62 @@
+#include "simdjson.h"
+#include "test_ondemand.h"
+
+using namespace simdjson;
+
+namespace document_stream_tests {
+
+    std::string my_string(ondemand::document& doc) {
+        std::stringstream ss;
+        ss << doc;
+        return ss.str();
+    }
```

Comment:
Is it possible that my_string actually consumes the document?

Reply:
I expect it does, yes.

Reply:
So when I changed your tests by adding to_string everywhere, I may have made it easier to pass in some sense. But it should be easy enough to add more tests.